### PR TITLE
[IE TESTS] Add Squeeze single layer test

### PR DESCRIPTION
**Pull request by iefode**

### CODE REVIEW

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/reshape_squeeze_reshape_relu.cpp [hunk 1]
```diff
@@ -31,19 +31,16 @@ namespace {
...
-    INSTANTIATE_TEST_CASE_P(reshape_squeeze_reshape_relu, ReshapeSqueezeReshapeRelu,
-                            ::testing::Combine(
-                                    ::testing::ValuesIn(inputs),
-                                    ::testing::ValuesIn(netPrecisions),
-                                    ::testing::Values(CommonTestUtils::DEVICE_CPU),
-                                    ::testing::Values(true)),
-                            ReshapeSqueezeReshapeRelu::getTestCaseName);
+    const std::vector<ngraph::helpers::SqueezeOpType> opTypes = {
+            ngraph::helpers::SqueezeOpType::SQUEEZE,
+            ngraph::helpers::SqueezeOpType::UNSQUEEZE
```
Comment (mikhail-treskin) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/reshape_squeeze_reshape_relu.cpp:36]: Why did you add additional unsqueeze case for certain subgraph which intended to be exactly reshape->squeeze->reshape?

File: inference-engine/tests/ngraph_functions/src/squeeze_unsqueeze.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace ngraph {
+namespace builder {
+std::shared_ptr<ngraph::Node> makeSqueezeUnsqueeze(const ngraph::Output<Node> &in,
+                                                   const element::Type &type,
+                                                   const std::vector<int> &squeeze_indices,
+                                                   ngraph::helpers::SqueezeOpType opType,
+                                                   bool isScalarAxes) {
+    ngraph::Shape shape = isScalarAxes ? ngraph::Shape{} : ngraph::Shape{squeeze_indices.size()};
+    auto constant = std::make_shared<ngraph::opset1::Constant>(type, shape, squeeze_indices);
```
Comment (mikhail-treskin) [inline inference-engine/tests/ngraph_functions/src/squeeze_unsqueeze.cpp:17]: I'm still wondering how it works when you pass `squeeze_indicces` e.g. {1,2} along with parameter isScalarAxes? Whether something works wrong or I don't understand how it's possible to construct such constant.

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/reshape_squeeze_reshape_relu.cpp (hunk 1 above)
Comment (iefode) [reply inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/reshape_squeeze_reshape_relu.cpp:36]: It's not additional test case, It was handles in test definition by flags.
